fdbserver/ClusterController.actor.cpp

```diff
@@ -4123,6 +6164,12 @@ void clusterRegisterMaster(ClusterControllerData* self, RegisterMasterRequest co
 	// Construct the client information
 	if (db->clientInfo->get().commitProxies != req.commitProxies ||
 	    db->clientInfo->get().grvProxies != req.grvProxies) {
+		TraceEvent("PublishNewClientInfo", self->id)
+		    .detail("Master", dbInfo.master.id().toString())
```

Comment:
ditto

Reply:
I believe logging details when new ClientInfo gets published is useful; at least I found it useful when triaging simulation failures. Further, the log line shouldn't cause too much spew as it gets recorded ONLY when new clientInfo is published. However, if log spew is a concern, am happy to downgrade logging level to SevDebug. 
Thanks.

Reply:
I mean this can be written as `.detail("Master", dbInfo.master.id())`.

Reply:
Got it. Done. 
Thanks.